ci/templatetags/seed.py

```diff
@@ -8,3 +8,15 @@
 @register.simple_tag
 def current_version():
     return seed_control_interface.__version__
+
+
+@register.filter
+def cleanup(value):
+    return value.replace('_', ' ')
```

Comment:
Possibly name this `unslug`? We're basically converting a slug version of something (`foo_bar`) to a normal version (`foo bar`).

We're also not guaranteed that these keys will always be in the form `foo_bar`, they might be `foo-bar`, or any other valid json, so maybe we should make it more generic, like changing any series of characters that are not alphanumeric into a single space.

We could do this with regex, an expression similar to `r'\W+'` will match all the things that we want to replace with a space. (We might also want to strip any whitespace from the beginning or end of the string).